agents/mcp_servers/deidentification/jsl-deid-mcp-server-v1/examples/langchain_example.py

```
@@ -0,0 +1,270 @@
+    """Lightweight MCP client for deidentification."""
+    
+    def __init__(self, url: str = "http://localhost:8000/mcp"):
+        self.url = url
+        self.session_id: Optional[str] = None
+        self.headers = {
+            "Content-Type": "application/json",
+            "Accept": "application/json, text/event-stream"
+        }
+        self._request_id = 0
+    
+    def _request(self, method: str, params: dict = None) -> dict:
```

Ruff (0.14.14)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)